build/openresty/patches/ngx_lua-0.10.21_02-dyn_upstream_keepalive.patch

```diff
@@ -673,7 +673,7 @@ index f71a3e00..0d403716 100644
 +
 +    items = (ngx_http_lua_balancer_keepalive_item_t *) (&upool->free + 1);
 +
-+    ngx_http_lua_assert((void *) items == ngx_align_ptr(items, NGX_ALIGNMENT));
++    /*ngx_http_lua_assert((void *) items == ngx_align_ptr(items, NGX_ALIGNMENT));*/
```

Comment:
We should remove this unneeded code, instead of commenting it out. It is a good habit to have.